source/documentation/archive/serving_data/fuseki1.md

```diff
@@ -0,0 +1,450 @@
+---
+title: "Fuseki: serving RDF data over HTTP"
+---
+
+## See the [Fuseki2 documentation](/documentation/fuseki2/).
+
+----
+> This page covers Fuseki v1.
+> Fuseki1 is deprecated and has been retired. 
+> The last release of Jena with this module is Jena 3.9.0.
+> See [jena-fuseki1/README.md](https://github.com/apache/jena/tree/master/fuseki1).
+> This is the original documentation.
+----
+
+Fuseki is a SPARQL server. It provides REST-style SPARQL HTTP Update, SPARQL Query,
+and SPARQL Update using the SPARQL protocol over HTTP.
+
+The relevant SPARQL standards are:
+
+-   [SPARQL 1.1 Query](http://www.w3.org/TR/sparql11-query/)
+-   [SPARQL 1.1 Update](http://www.w3.org/TR/sparql11-update/)
+-   [SPARQL 1.1 Protocol](http://www.w3.org/TR/sparql11-protocol/)
+-   [SPARQL 1.1 Graph Store HTTP Protocol](http://www.w3.org/TR/sparql11-http-rdf-update/)
+
+## Contents
+
+-   [Download](#download-fuseki1)
+-   [Getting Started](#getting-started-with-fuseki)
+-   [Security](#security-and-access-control)
+-   [Logging](#logging)
+-   [Server URI scheme](#server-uri-scheme)
+-   [Running a Fuseki Server](#running-a-fuseki-server)
+-   [Fuseki Configuration File](#fuseki-configuration-file)
+-   [SPARQL Over HTTP](#sparql-over-http)
+-   [Use from Java](#use-from-java)
+-   [Development System](#development-system)
+
+## Download Fuseki1
+
+Binaries for Fuseki1 are available from the 
+[maven
+repositories](http://central.maven.org/maven2/org/apache/jena/jena-fuseki1/).
```

Comment:
I think they removed the old central.maven.org sites (not sure why). This one works for me: `https://search.maven.org/artifact/org.apache.jena/jena-fuseki1`